DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mlr3extralearners
 Title: Extra Learners For mlr3
-Version: 0.6.1-9000
+Version: 0.6.2-9000
```

Comment:
```suggestion
Version: 0.6.1-9000
```

Reply:
I will increment to 0.6.2 when I make a new release

Reply:
That's fine, I would suggest though that you remove the checklist item that requests all PRs to increment the version number, which I why I made this change.

"[ ] Updated the package version (For patches (bug fixing, small added feature): add 1 to the 3rd number. For minor updates (new learner, new features): add 1 to the 2nd number)."

Reply:
Thanks, this is indeed outdated!